src/status_im/contexts/wallet/send/events_test.cljs

```diff
@@ -163,3 +174,283 @@
+                            :recipient   {:recipient-type :saved-address
+                                          :label          "label"}
+                            :other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
+
+(h/deftest-event :wallet/clean-send-amount
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:amount      10
+                            :other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
+
+(h/deftest-event :wallet/clean-disabled-from-networks
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                            :other-props             :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
+
+(h/deftest-event :wallet/clean-from-locked-amounts
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:from-locked-amounts "value"
+                            :other-props         :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
+
+(h/deftest-event :wallet/disable-from-networks
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                                          :other-props             :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id [:optimism]]))))))
+
+(h/deftest-event :wallet/unlock-from-amount
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props         :value
+                                          :from-locked-amounts {}}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props         :value
+                            :from-locked-amounts {:chain-id [1 10]}}}}})
+    (is (match? expected-db (:db (dispatch [event-id :chain-id]))))))
+
+(h/deftest-event :wallet/lock-from-amount
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props         :value
+                                          :from-locked-amounts {:amount 10}}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id :amount 10]))))))
+
+(h/deftest-event :wallet/clean-selected-token
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props        :value
+                            :token              "ETH"
+                            :token-display-name "ETH"
+                            :tx-type            :tx/collectible-erc-721}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
```

Comment:
@mmilad75, isn't this a case when `match?` will return `true` even if keys weren't cleared from the db?

Reply:
Yes, but I tested with `=` before converting it to `match?` so I made sure it's not the case here.

Reply:
@mmilad75, if I understand correctly `match?` won't be able to detect if keys were removed or not. It will only detect that `:other-props` key survived. And the goal of the test is to check that keys were properly cleared. So there is no point in using `match?`. 
The fact that it worked with `=` is not relevant because our goal is to cover future event changes. If someone alter the event lately and it won't delete keys anymore, the test should fail. And it won't fail with `match?`.

Reply:
So what do you suggest here?

Reply:
You can use `get-in` to get particular keys and ensure they are `nil` after event invocation.

Reply:
Nice. I will do so and update the PR

Reply:
Done